### ISSUE TITLE
[pfe-select] Triangle icon not registering click event

### ISSUE THREAD

**Issue by KamiQuasi** [patternfly/patternfly-elements#1801]
## Description of the issue
The triangle icon does not function as a clickable part of the select component. This means that if someone clicks on the icon that the select drop-down will not expand. 

### Impacted component(s)

- pfe-select

### Steps to reproduce

1. Go to https://patternflyelements.com/components/select/
2. Click on the triangle icon on any of the demo components
3. Note that the component does not activate

### Expected behavior

The icon should be a clickable option for mouse users to expand and collapse the select component.

**Comment by bennypowers** [comment]
@KamiQuasi looks like the triangle has disappeared entirely
